google/cloud/bigtable/examples/run_examples_utils.sh

```diff
@@ -72,97 +131,48 @@ function run_all_instance_admin_examples {
   shift 2
 
   local setenv="env"
```

Comment:
This line no longer necessary?

Reply:
Yes. After we stopped running the instance admin integration tests in production this is no longer needed.  All the tests in a group (admin, data, instance admin) use the same environment variables.

Reply:
Doh! You meant that I could remove that line... yes, I can.  Will do in the next PR.

Reply:
Sorry for the subtlety; yes, I did mean that if you don't need it, feel free to remove it. 😃